### ISSUE TITLE
NeoCSV link in the README is broken

### ISSUE THREAD

**Issue by jordanmontt** [svenvc/NeoCSV#31]
If you click on it, it will take you to a non-existing page

**Comment by svenvc** [comment]
true, but that is a problem with the people who publish the book

**Comment by jordanmontt** [comment]
Yes, maybe removing the link or putting another link can be a solution?

**Comment by svenvc** [comment]
I helped write the book, I would love to link to individual chapters, I raised this point before, to no avail.

Maybe it help when you do it, in the mailing list or on discord ?

**Comment by jordanmontt** [comment]
I understand and I read the book :) I am only saying that one link in the README is not working so maybe is better to fix it because it can confuse some people

**Comment by svenvc** [comment]
I posted this question on the ML:

https://lists.pharo.org/empathy/thread/LDBOGTUA3KZPLM4AL5DNESIH3RNEZOJR

**Comment by labordep** [comment]
Is the content of this link ? 

https://book.huihoo.com/smalltalk/pharo/enterprise-pharo/book-result/NeoCSV/NeoCSV.html

**Comment by svenvc** [comment]
OK, it works, that is indeed how it used to work.

But what is this site and who maintains it ?

**Comment by labordep** [comment]
I don't know, I just want to know if this is the content of the dead link for read it :-)

**Comment by svenvc** [comment]
Yes it seems so.
